requirements.txt

```diff
@@ -1,8 +1,3 @@
 vantage6
```

Comment:
If you're editing stuff here anyway, you might as well add versions for your requirements.